test/functional/fixtures/native-dialogs-handling/pages/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Index</title>
+</head>
+<body>
+<div id='confirmResult'></div>
+<div id='promptResult'></div>
+<a href="confirm.html">Confirm page</a>
+<a href="index.html">This page</a>
+<a id="fakeLink" href="javascript:alert()">Fake link</a>
+<button id="confirmBtn" onclick="onConfirmBtnClick()">Confirm
+</button>
+<button id="promptBtn" onclick="window.prompt('Prompt dialog')">Prompt
+</button>
+<button id="dialogsBtn" onclick="onDialogsButtonClick()">Dialogs</button>
+<button id="simpleBtn">Button</button>
+<button id="alertBtn" onclick="alert('alert button clicked')">alert</button>

```

Comment:
`onclick="window.prompt('Prompt dialog')"`
`onclick="alert('alert button clicked')"`

use the same approach in similar places.

`onclick="window.prompt('Prompt dialog')"`
`onclick="window.alert('Alert dialog')"`